docs/components/Readme.md

```diff
@@ -0,0 +1,33 @@
+[Readme](../../README.md) » BaseComponents
+
+# FSXABaseSection
+
+1. [Introduction](#introduction)
+2. [Components](#components)
+
+## Introduction
+
+The [FSXA-Pattern-Library](https://github.com/e-Spirit/fsxa-pattern-library) provides multiple base components, which come with a lot attributes and methods to help you build your PWA. <br/>
+This chapter will introduce you to these different components and describes when and how to use them.
+
```

Comment:
different tenses; use either 'introduces' or 'and will describe'. I prefer the former.